src/styles/main.scss

```
@@ -1,3 +1,40 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+
 body {
   margin: 0;
 }
+
+.calendar {
+  width: $calendar-size;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: $padding-calendar;
+  gap: $cell-distance;
+
+  &__cell {
+    @include square ($cell-size);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
+    box-sizing: border-box;
+    font-size: $font-size;
+    transition: 0.5s;
+
```

```suggestion
    transition: 0.5s;
```
don't animate all properties, animate only you need (transform, background-color)

---

done)